Mathlib/Analysis/Normed/Module/FiniteDimension.lean

```diff
@@ -269,6 +269,21 @@ theorem isOpen_setOf_nat_le_rank (n : ℕ) :
     continuous_pi fun x => (ContinuousLinearMap.apply 𝕜 F (x : E)).continuous
   exact isOpen_setOf_linearIndependent.preimage this
 
+theorem isOpen_setOf_affineIndependent {ι : Type*} [Finite ι] :
+    IsOpen {p : ι → E | AffineIndependent 𝕜 p} := by
+  classical
+  rcases isEmpty_or_nonempty ι with h | ⟨⟨i₀⟩⟩
+  · exact isOpen_discrete _
+  · simp_rw [affineIndependent_iff_linearIndependent_vsub 𝕜 _ i₀]
+    let ι' := { x // x ≠ i₀ }
+    cases nonempty_fintype ι
+    haveI : Fintype ι' := Subtype.fintype _
+    convert_to
+      IsOpen ((fun (p : ι → E) (i : ι') ↦ p i -ᵥ p i₀) ⁻¹' {p : ι' → E | LinearIndependent 𝕜 p})
+    refine isOpen_setOf_linearIndependent.preimage ?_
+    exact continuous_pi fun i' ↦
```

Comment:
Can `fun_prop` prove this goal? If no, please either add missing `@[fun_prop]`s or add a `TODO`.
Otherwise LGTM. Thanks!
bors d+